documents/vci-flow.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+
+header VCI flow
+title
+VCI flow
+A scenario for receiving a credential in your SSI wallet
+end title
+
+autonumber
+
+participant "User" as user order 0 #ORANGE
+participant "Wallet" as wallet order 1 #RED
+participant "Demo app" as demo order 2 #BLUE
+participant "Agent" as agent order 3 #GREEN
+
+user -> demo: from the list, selects a credential type that she wants to receive
+demo -> demo: generates a form with the form data in ecosystem config file
+demo -> user: shows user the form to user.
```

Comment:
No proper English

Reply:
changed to `The demo application displays this form to the user.`